README.md

```diff
@@ -1,34 +1,65 @@
 # SSA
```

Comment:
would be nice to have more context about what the project is

Reply:
@goens got a good summary? (: